### ISSUE TITLE
Support for ES6/7 (Babel.js)

### ISSUE THREAD

**Issue by jeffijoe** [20Tauri/DoxyDoxygen#13]
Hey, great plugin!

Was wondering when you will support ES6/7 code? Example:

``` javascript
/**
 * The InvitationController.
 */
class InvitationController extends Mn.Object {
  /**
   * Fetches and shows the invitation.

   * @param  {String} invitationSubject The invitation subject identifier.
   * @param  {String} invitationCode    The invitation code.
   */
  showInvitation(invitationSubject, invitationCode) {
    // .. do stuff
  }
}
```

**Comment by 20Tauri** [comment]
Some ES6/7 features should already be supported:
- arrow functions
- generators
- class (without "extends")

"get" / "set", "constructor", non ascii identifiers or identifiers with \u is now supported in current release (0.16.0)
I you have others problems with JavaScript support simply report them, I will update the plugin

**Comment by 20Tauri** [comment]
Fixed since 0.16.0
I you have others problems with JavaScript support simply report them, I will update the plugin
